### DISCUSSION TITLE
[委托]旧文明战斗机图纸收集

### DISCUSSION THREAD

**Discussion by GrunderIndustries** [share121/inter-knot#2506] [其他]
![image](https://i0.hdslb.com/bfs/article/4cbd77354be07d9347fd46145640d7e3cb7aebd7.jpg@1256w_944h_!web-article-pic.avif)
鄙人对旧文明的战斗机非常好奇 打算弄点图纸
遂发委托 一份战斗机图纸5000丁尼 如果是网上找不到的未知型号追加100菲林(类似于图片这种)
如果能找到战斗机残骸再加320菲林

**Comment by qwq775** [comment]
这个委托我接了！

**Comment by nhb-bcd** [comment]
旧时期战斗机展示视频要吗
<img src="https://img-static.mihoyo.com/communityweb/upload/6b88559cbba96a1fa965e5412736370d.png" alt="兔躯一震" width="75" >

**Comment by Wangsheng-Zhongli** [comment]
此委托在下颇感兴趣，已决定接下。恰好，在下手中有一份美F15战斗机的图纸，保留较为完整，若阁下对此有意，愿意将其转交。不知这份图纸是否符合阁下的期望？如有其他需求，也可详谈。
![fj3](https://github.com/user-attachments/assets/a84602ef-2da7-4083-8eb9-12466dc7f003)
![fj5](https://github.com/user-attachments/assets/3dc733e5-b41c-42b9-9d16-2510f02e38f1)
![fj4](https://github.com/user-attachments/assets/9ce46912-b8b3-41a4-961c-a969e600793b)
![fj2](https://github.com/user-attachments/assets/0bb7276e-1564-4cc8-9249-05cd8903b93d)
![fj1](https://github.com/user-attachments/assets/fae87104-ee0f-4cd2-8d9e-6a1da7123170)

**Reply by GrunderIndustries** [reply]
25000丁尼收了
